prow/config/jobs/release-builder.yaml

```diff
@@ -1,12 +1,12 @@
 org: istio
 repo: release-builder
-image: gcr.io/istio-testing/build-tools:2019-09-23T12-48-14
+image: gcr.io/istio-testing/build-tools:2019-09-29T15-31-13
 suppress_entrypoint: true
 branches:
-- master
+  - master
 
 jobs:
-- name: lint
-  command: [make, lint]
-- name: test
-  command: [make, test]
+  - name: lint
```

Comment:
does yamllint support this check and we don't have it turned on? I don't mind either way but don't want commits flip flopping between indented and not (or our docs!)

Reply:
I was surprised to find those two files not formatted in a standard way.

We do yaml linting in this repo, but we do currently have a lot of the yaml style linting options turned off. Cranking up those options is on my todo list once we're fully on the new build-tools plan everywhere